app-office/libreoffice/libreoffice-9999.ebuild

```diff
@@ -507,7 +505,6 @@ src_configure() {
 		--without-system-jfreereport
 		--without-system_apache_commons
```

Comment:
https://bugs.gentoo.org/793776
https://github.com/LibreOffice/core/commit/6de0b1710adfba82c96b75a5da6f52633a54c692
"Get rid of apache-commons-logging"